src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSBatchSink.java

```diff
@@ -67,6 +67,7 @@
   private static final String RECORDS_UPDATED_METRIC = "records.updated";
   public static final String AVRO_NAMED_OUTPUT = "avro.mo.config.namedOutput";
   public static final String COMMON_NAMED_OUTPUT = "mapreduce.output.basename";
+  public static final String CONTENT_TYPE = "Content-Type";
```

Comment:
Prefix the FS configuration key. Also, use "." notation. E.g. "io.cdap.gcs.batch.sink.content.type".

Reply:
This was changed to `public static final String CONTENT_TYPE = "io.cdap.gcs.batch.sink.content.type";` in the updated PR.